src/routes/(console)/organization-[organization]/billing/planSummary.svelte

```diff
@@ -239,3 +253,10 @@
         </svelte:fragment>
     </CardGrid>
 {/if}
+
+<style>
+    .loader-container {
+        flex-grow: 1;
```

Comment:
do we typically use the `u-stretch` class for this?

Reply:
Yes! Thanks! I tried to find this one in the old pink docs, but couldn't find it that quickly. Need to familiarise myself a bit more with those utility classes.